README.md

```diff
@@ -9,12 +9,13 @@ was jQuery special events only) – jQuery is no longer a requirement. However,
 if you do not have jQuery you will require a polyfill for Object.assign to
 support older browsers. I can recommend <a href="https://github.com/cruncher/object.assign">Object.assign polyfill</a> :)
 
-##Demo and docs
+## Demo and docs
 
 <a href="http://stephband.info/jquery.event.move/">stephband.info/jquery.event.move/</a>
 
-##Move events
+## Move events
 
+```html
```

Comment:
This isn't supposed to be highlighted as HTML. It's a definition list of move events.

Reply:
Sorry :facepalm: 
fixed